src/routes/oauth/utils.ts

```diff
@@ -59,14 +59,26 @@ export const transformOauthProfile = async (
 
   const emailVerified = !!normalised.emailVerified;
 
+  let allowedRoles: string[] =  ENV.AUTH_USER_DEFAULT_ALLOWED_ROLES;
+
+  if(options?.allowedRoles){
```

Comment:
Could you please run prettier on this code? It looks like the formatting is incorrect.

Reply:
done @szilarddoro